proxy.go

```diff
@@ -188,6 +216,207 @@ func (proxy *proxy) releaseToken(token Token) (*tokenInfo, error) {
 	return info, nil
 }
 
+var storeStateDir = "/var/lib/clear-containers/proxy/"
```

Comment:
Can you move this to the top of the file? Its value should also be passed in from `Makefile` so that at the top it's something like:

```
// XXX: set by build
var storeStateDir string
```

Reply:
Got it. Will do. Should it be set up in Makefile to "/var/lib/clear-containers/proxy/"?